docs/sources/flow/reference/components/otelcol.exporter.jaeger.md

```diff
@@ -78,6 +78,7 @@ Name | Type | Description | Default | Required
 `wait_for_ready` | `boolean` | Waits for gRPC connection to be in the `READY` state before sending data. | `false` | no
 `headers` | `map(string)` | Additional headers to send with the request. | `{}` | no
 `balancer_name` | `string` | Which gRPC client-side load balancer to use for requests. | `pick_first` | no
+`authority` | `string` | Overrides the default `:authority` header in gRPC requests from the gRPC client. | | no
```

Comment:
looking at https://github.com/grpc/grpc-go/blob/master/clientconn_authority_test.go, the default authority is the address you are connecting to

Reply:
It would be nice to document it in a similar way to how `balancer_name` is documented - check out this file for an example:
`agent/docs/sources/shared/flow/reference/components/otelcol-grpc-balancer-name.md`